src/main/java/com/mastercard/dis/mids/reference/MIDSReferenceApplication.java

```diff
@@ -149,9 +151,13 @@ void handleOption(String option) {
 
 
 
-    private void performMultiDocEnrollment() {
+     void performMultiDocEnrollment() {
         try {
             log.info("<<--- MultiDocEnrollment Started --->>");
+            Cache.pdsMultiDocument = midsReference.getPDS(false, Arrays.asList("facePDS", "attributePDS", "evidencePDS"));
+            System.out.println("Enter the workflowId multiDoc");
+            Constants.MULTI_DOCUMENT_WORKFLOW_ID = scanner.nextLine();
```

Comment:
I think, this constant variable is misused.
Please, read this: https://www.scaler.com/topics/constant-in-java/

I understand that this project is just a reference code, but we should keep the good practices.

Reply:
I cannot use the final because the variable may not be initialized.